app/models/patron_request.rb

```diff
@@ -146,6 +151,10 @@ def earliest_delivery_estimate(scan: false)
     { 'date' => Time.zone.today, 'display_date' => 'No date/time estimate' }
   end
 
+  def title_request_expiration_date
+    Time.zone.today + 1.year
+  end
+
```

Comment:
don't think you need this. Think you need to change earliest_delievery_estiamate to 
`if title_only? || any_items_avaliable?`

Reply:
I think I'll update earliest_delivery_estimate to handle no items and to return a year from now, b/c otherwise they return today's date.  I can then get rid of the title_request_expiration_date method

Reply:
I changed the earliest delivery estimate to handle the title only condition, b/c the existing conditions would just return today's date, and we want to have a year from now as the date .